ptbcontrib/postgres_persistence/README.md

```diff
@@ -0,0 +1,35 @@
+# Persistence class with Postgresql database as backend.
```

Comment:
I think codacy is actually pretty clear this time. here it complains about the trailing dot at headings.
```suggestion
# Persistence class with Postgresql database as backend
```